go/vt/vtctl/reparentutil/replication_test.go

```diff
@@ -161,15 +161,15 @@ func TestFindValidEmergencyReparentCandidates(t *testing.T) {
 			shouldErr: false,
 		},
 		{
-			name: "tablet with errant GTIDs is excluded",
+			name: "tablet with superset GTIDs is included",
 			statusMap: map[string]*replicationdatapb.StopReplicationStatus{
 				"r1": {
 					After: &replicationdatapb.Status{
 						SourceUuid:       "3E11FA47-71CA-11E1-9E33-C80AA9429562",
 						RelayLogPosition: "MySQL56/3E11FA47-71CA-11E1-9E33-C80AA9429562:1-5",
 					},
 				},
-				"errant": {
+				"r2": {
```

Comment:
It's not clear to me how r2 does not have an errant GTID in this case. Isn't the test comparing the replica positions against the primary? r2 does in fact have a GTID that the primary does not have. Maybe I'm misunderstanding the test here though... should we be passing in a nil/empty primaryStatusMap for the ERS case?

Reply:
primaryStatusMap has no effect on how we compute valid candidates. Its contents are simply stuffed into the positionMap if it is non-empty. The way the test is written does make it confusing.

Reply:
I'll remove it from these tests so that it is less confusing. One of the things we need to revisit is whether when primaryStatusMap is non-empty we should be using that to determine errant GTIDs.